npmrc

```diff
@@ -0,0 +1 @@
+registry=https://cli-npm.heroku.com
```

Comment:
I think right now, no. `cli-npm.heroku.com` is a cloudfront server. It might be better in this case though to connect to the dyno directly. The build is running in us-east (usually) so it might make sense to use that instead.

But if we wanted to do that, we might want to set up elephants in each region (which wouldn't be hard to do).

I think we should leave it as cloudfront for now with this endpoint, then we want to do some performance testing to see if clousfront is better or worse.